src/AccessibilityInsights.SharedUx/Dialogs/StartUpModeControl.xaml.cs

```diff
@@ -61,7 +61,7 @@ public StartUpModeControl()
             Uri releaseNotesUri = MsiUtilities.GetReleaseNotesUri(new SetupExceptionReporter());
             if (releaseNotesUri != null)
             {
-                hlLink.NavigateUri = releaseNotesUri;
+                //hlLink.NavigateUri = releaseNotesUri;
```

Comment:
should the `if` be remove now that the `then` block is empty? or maybe the commented line was not intentional?

Reply:
Good catch! Removed this now unused code.